docs/source/models/other_models/alzheimers_population/index.rst

```diff
@@ -30,4 +30,230 @@
+mean :math:`\lambda_{g,t} \cdot \Delta t \cdot 1_{\{\text{simulation
+step times}\}}(t)`, where :math:`\lambda_{g,t}` is the entrance rate of
+new simulants (measured in count of simulants per unit time) at time
+:math:`t`, :math:`\Delta t` is the length of a simulation time step, and
+:math:`1_A` is the indicator function of the set :math:`A` (the
+indicator function zeros out the entrance rate at times when the
+simulation is not taking a step). Our goal is to determine the entrance
+rate :math:`\lambda_{g,t}` for each :math:`g` and :math:`t`.
+
+Let :math:`A_g(t)` be the cumulative number of incident cases of AD by
+time :math:`t` in demographic group :math:`g` in the real population.
+Since our simulation is scaled down by a factor of :math:`S`, the rate
+at which we want to add simulants is
+
+.. math::
+
+  \lambda_{g,t} = S \cdot A_g'(t).
+
+We rewrite this in terms of quantities that we can estimate from the
+available data:
+
+.. math::
+  :label: AD_entrance_rate_eq
+
+  \lambda_{g,t}
+  = S \cdot A_g'(t)
+  = S \cdot \frac{A_g'(t)}{\hat X^\text{real}_{g,t}}
+    \cdot \hat X^\text{real}_{g,t}
+  = S \cdot i_{g,t} \cdot \hat X^\text{real}_{g,t},
+
+where :math:`i_{g,t} = A_g'(t) /\hat X^\text{real}_{g,t}` is the **total
+population incidence hazard** of AD in demographic group :math:`g` at
+time :math:`t`. We know the model scale :math:`S` from
+:eq:`model_scale_eq` above, and we can estimate the quantities
+:math:`i_{g,t}` and :math:`X^\text{real}_{g,t}` from GBD as follows.
+
+Let :math:`y(t)` denote the year in which time :math:`t` occurs. If we
+assume that the hazard :math:`i_{g,t}` is constant throughout the year
+:math:`y(t)`, then it is equal to its person-time-average over the year,
+which is the **total population incidence rate**:
+
+.. math::
+
+  i_{g,t}
+  = \frac{\text{# of incident cases of AD in group $g$ in year $y(t)$}}
+    {\text{total person-years in group $g$ in year $y(t)$}}.
+
+This is the raw AD incidence rate we pull from GBD (*not* the susceptible
+population incidence rate usually calculated by Vivarium Inputs).
+If we assume that the population :math:`\hat X^\text{real}_{g,t}` is
+constant throughout the year :math:`y(t)`, then it is equal to its
+time-average over the year:
+
+.. math::
+
+  \hat X^\text{real}_{g,t}
+  = \text{average population in group $g$ during the year $y(t)$}.
+
+This is the population we pull from GBD using get_population. Thus,
+:eq:`AD_entrance_rate_eq` expresses the entrance rate
+:math:`\lambda_{g,t}` in terms of quantities we can estimate from data.
+
+.. note::
+
+  Based on plots from GBD compare, we will make the simplifying
+  assumption that for each demographic group :math:`g`, the Alzheimer's
+  incidence rate :math:`i_{g,t}` does not change over time. Thus, we
+  will use GBD 2021 data and assume that :math:`i_{g,t}` equals the AD
+  incidence rate in 2021 from for all times :math:`t`.
+
+  For Model 2 of the Alzheimer's simulation, we will use GBD 2021 data
```

Comment:
Should tables about specific data sources go in this doc? Or maybe this doc can link to the data tables in the cause model doc

Reply:
I'm not sure...